pkg/kubelet/kubelet_network.go

```diff
@@ -121,7 +121,7 @@ func (kl *Kubelet) providerRequiresNetworkingConfiguration() bool {
 	// is used or whether we are using overlay networking. We should return
 	// true for cloud providers if they implement Routes() interface and
 	// we are not using overlay networking.
-	if kl.cloud == nil || kl.cloud.ProviderName() != "gce" || kl.flannelExperimentalOverlay {
+	if kl.cloud == nil || kl.cloud.ProviderName() != "gce" {
```

Comment:
One can use the CNI flannel plugin. But that needs extra setup for flannel daemon and etc. 

I think that much cleaner than having a specific flag for flannel.